### PR TITLE
Tom meet the team

### PR DESCRIPTION
**Pull request by antoinepetty**
First draft of this blog post

image

### CODE REVIEW

**Comment by BK-211** [comment]
Intro should say "PSC" rather than "CLY"?

**Comment by TomWheal** [comment]
I can confirm I went to PSC not CLY. Also might want to crop the photo of me?

**Comment by antoinepetty** [comment]
Not sure if this photo is a bit pixelated now that it's cropped

File: _posts/2022-01-21-meet-the-team-tom-wheal.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+           src="/images/content/blog/meet-the-team/tom.jpg"
+           caption="Tom Wheal" %}
+
+#### How did you first get involved in (competing with) Student Robotics?
+
+I was a competitor in Student robotics in 2014 and 2015, while I was at Peter Symonds College, Winchester (PSC). During both years I was involved with all aspects of the robot build, alongside running our social media accounts and moved on to lead the team in 2015. After starting at the University of Southampton, I volunteered to help out at the competition. 
+
+#### What do you do as a volunteer for Student Robotics?
+
+I currently mentor 2 teams helping them get the most out of Student Robotics. Alongside this, I am involved in the Kit Team, which I was on the committee for previously and helped with the charity re-organisation in 2019.
```
Comment (antoinepetty) [inline _posts/2022-01-21-meet-the-team-tom-wheal.md:21]: Have requested a bit more detail about the kit team here

**Comment by raccube** [comment]
Looks very pixelated on HiDPI screens (Pixel Ratio: 3)

Tested on iPhone 12 & 2021 MacBook Pro

<details><summary>Laptop screenshot</summary>
Screenshot 2022-01-23 at 12 55 54</details>

<details><summary>Phone screenshot</summary>
IMG_3A232F7A13ED-1
</details>

**Review by sedders123 — APPROVED** [review]

**Pull request merged**